buildbot/osuosl/master/config/status.py

```diff
@@ -380,4 +380,11 @@ def getReporters():
             subject = "Build %(builder)s Failure",
             mode = "failing",
             builders = ["clang-cmake-x86_64-avx512-linux"]),
+        reporters.MailNotifier(
+            fromaddr = "llvm.buildmaster@lab.llvm.org",
+            sendToInterestedUsers = False,
+            messageFormatter = LLVMInformativeMailNotifier, # TODO: remove after moving to prod.
```

Comment:
Could you elaborate what this TODO comment means, please?

Reply:
in instructions it is said:

> It is possible to allow email addresses to unconditionally receive notifications on build failure; for this you’ll need to add an InformativeMailNotifier to buildbot/osuosl/master/config/status.py. This is particularly useful for the staging buildmaster which is silent otherwise.

So I thought it's a good idea to have it but only for the "staging" phase. So I am planning to replace notification settings with a "common" one afterwards. Can drop this todo altogether if that helps.

Reply:
removed and added a note to myself